iocBoot/ioc-ArbiterPLC/Makefile

```
@@ -1,4 +1,4 @@
-IOC_TOP = /reg/g/pcds/epics/ioc/common/ads-ioc/R0.3.1
+IOC_TOP = /reg/g/pcds/epics/ioc/common/ads-ioc/R0.6.0
```

Current version is R0.6.1
```suggestion
IOC_TOP = /reg/g/pcds/epics/ioc/common/ads-ioc/R0.6.1
```

---

I agree. fixed.